include/psp2/kernel/clib.h

```diff
@@ -51,6 +66,22 @@ int sceClibMemcmp(const void *s1, const void *s2, SceSize len);
 
 void *sceClibMemchr(const void *src, int ch, SceSize len);
 
+SceClibMspace sceClibMspaceCreate(void* memBlockAddr, SceSize size);
```

Comment:
```suggestion
SceClibMspace sceClibMspaceCreate(void* memblock_addr, SceSize size);
```
or
```suggestion
SceClibMspace sceClibMspaceCreate(void* mem_block_addr, SceSize size);
```

but how about just `memblock`?

Reply:
Requested adjustments have been made.